eZ/Publish/API/Repository/Values/Content/Query/Criterion/Operator.php

```diff
@@ -24,4 +24,22 @@
     const BETWEEN = 'between';
     const LIKE = 'like';
     const CONTAINS = 'contains';
+
+    /**
+     * @return array
+     */
+    public static function getAvaliableOperators()
```

Comment:
afaik this does not have any meaning in the PHP API, as operators are never referred to by string names like "EQ"  and so on.